meta-client/src/test/java/org/apache/helix/metaclient/impl/zk/TestZkMetaClient.java

```diff
@@ -41,8 +53,125 @@ public void prepare() {
   }
 
   @Test
-  public void dummyTest() {
-    Assert.assertEquals(1+1, 2);
+  public void testGet() {
+    final String key = "/TestZkMetaClient_testGet";
+    ZkMetaClient<String> zkMetaClient = createZkMetaClient();
+    String value;
+    zkMetaClient.create(key, ENTRY_STRING_VALUE);
```

Comment:
Following the connect() comment above, after the full implementation, metaclient should call connect() before any operation, right?

Reply:
yes. connect() is needed before the first operation.